open_prices/api/proofs/serializers.py

```diff
@@ -18,6 +32,7 @@ class Meta:
 
 class ProofFullSerializer(ProofSerializer):
     location = LocationSerializer()
+    predictions = ProofPredictionSerializer(many=True, read_only=True)
```

Comment:
Not sure we should return the predictions when fetching a "list" of proofs. It could be returned only in the "retrieve" endpoint

Reply:
Indeed, we could restrict it to the retrieve endpoint!

Reply:
done here: https://github.com/openfoodfacts/open-prices/pull/605